inc/views/base/modcp/finduser.twig

```diff
@@ -0,0 +1,125 @@
+{% extends 'layouts/master.twig' %}
+
+{% block head %}
+    <title>{{ mybb.settings.bbname }} - {{ lang.find_users }}</title>
+{% endblock head %}
+
+{% block body %}
+    <table width="100%" border="0" align="center">
+        <tr>
+            {# Modcp nav #}
+            <td valign="top">
+                <table border="0" cellspacing="{{ theme.borderwidth }}" cellpadding="{{ theme.tablespace }}" class="tborder">
+                    <tr>
+                        <td class="thead" colspan="5"><strong>{{ lang.users }}</strong></td>
+                    </tr>
+                    <tr>
+                        <td class="tcat" width="30%"><span class="smalltext"><strong>{{ lang.username }}</strong></span></td>
+                        <td class="tcat" align="center" width="15%"><span class="smalltext"><strong>{{ lang.usergroup }}</strong></span></td>
+                        <td class="tcat" align="center" width="15%"><span class="smalltext"><strong>{{ lang.regdate }}</strong></span></td>
+                        <td class="tcat" align="center" width="15%"><span class="smalltext"><strong>{{ lang.lastvisit }}</strong></span></td>
+                        <td class="tcat" align="center" width="10%"><span class="smalltext"><strong>{{ lang.postnum }}</strong></span></td>
+                    </tr>
+                    {% if users %}
```

Comment:
I believe this `if` is redundant. if we needed an else clause for an empty array, we can use:

```twig
{% for user in users %}
    {# user #}
{% else %}
    {# no users #}
{% endfor %}
```

Reply:
Fixed, and added no_user row back (somehow it didn't make it in the first commit).